tests/test_units/test_dataio.py

```diff
@@ -37,17 +37,15 @@ def test_generate_metadata_simple(globalconfig1):
 def test_missing_or_wrong_config_exports_with_warning(regsurf):
     """In case a config is missing, or is invalid, do export with warning."""
 
-    with pytest.warns(
-        PendingDeprecationWarning, match="One or more keys required for valid metadata"
-    ):
+    with pytest.warns(PendingDeprecationWarning):
         edata = ExportData(config={}, content="depth")
 
-    with pytest.warns(PendingDeprecationWarning, match="One or more"):
+    with pytest.warns(PendingDeprecationWarning):
```

Comment:
IMO, no need to check these warnings strings anymore. I think its fine to relay on pydantic to produce these warnings.